run.py

```diff
@@ -164,16 +165,21 @@ def register_node():
         logger.info("Server response did not include heartbeat, using default {}".format(heartbeat_interval))
 
 
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("token", help="Broadway cluster token")
+    parser.add_argument("worker_id", metavar="worker-id", help="Unique worker id for registration")
+    return parser.parse_args()
+
+
 if __name__ == "__main__":
     # check valid usage
-    if len(sys.argv) != 3:
-        print_usage()
-        exit(-1)
+    args = parse_args()
 
     signal.signal(signal.SIGINT, signal_handler)
 
-    token = sys.argv[1]
-    worker_id = sys.argv[2]
+    token = args.token
```

Comment:
Being really picky but no need to create new variables for `token` and `worker_id` here since we can just use `args.token` and `args.worker_id` everywhere. Does not improve readability.

Reply:
Ah ok. We do need worker_id though since it's a global variable

Reply:
`args` is global right? so we can just use it?

Reply:
yes.. But I feel like we shouldn't really assume worker_id is always args.worker_id

Reply:
Oh, I see you have already had a global `worker_id`. I guess its fine then